servers/rendering/shader_language.cpp

```diff
@@ -3479,6 +3479,9 @@ bool ShaderLanguage::_validate_function_call(BlockNode *p_block, const FunctionI
 				}
 			}
 
+			if (r_is_custom_function) {
```

Comment:
Won't this always be false?

Reply:
It's working when the pointer is not a null.

Reply:
Ah, that makes sense